po/ko.po

```diff
@@ -3010,11 +3137,11 @@ msgstr "서비스 구현"
 
 #: src/types-and-values/exercise.md:16
 msgid "// The recursive case.\n"
-msgstr ""
+msgstr "// 재귀 사례입니다.\n"
 
 #: src/types-and-values/exercise.md:23 src/types-and-values/solution.md:14
 msgid "\"fib(n) = {}\""
-msgstr ""
+msgstr "'fib(n) = {}'"
```

Comment:
This and many other places, it changes double quotes to single quotes.  I think it's a program text and this change will make build errors.  I could be wrong but can you please check this?

Reply:
I'll check it out, thanks.